source/_posts/test.bib

```diff
@@ -0,0 +1,35 @@
+@article{article,
+  author  = {Peter Adams},
+  title   = {The title of the work},
+  journal = {The name of the journal},
+  year    = 1993,
+  number  = 2,
+  pages   = {201-213},
+  month   = 7,
+  note    = {An optional note},
+  volume  = 4
+}
+
+@book{book,
+  author    = {Peter Babington},
+  title     = {The title of the work},
+  publisher = {The name of the publisher},
+  year      = 1993,
+  volume    = 4,
+  series    = 10,
+  address   = {The address},
+  edition   = 3,
+  month     = 7,
+  note      = {An optional note},
+  isbn      = {3257227892}
+}
+
+@booklet{booklet,
+  title        = {The title of the work},
+  author       = {Peter Caxton},
+  howpublished = {How it was published},
+  address      = {The address of the publisher},
+  month        = 7,
+  year         = 1993,
+  note         = {An optional note}
+}
```

Comment:
I'd have a code chunk in the Rmd document that generates some bib entries automatically, e.g., 

````
```{r, include=FALSE}
knitr::write_bib(c('base', 'blogdown'), 'test.bib')
```
````